readme.txt

```diff
@@ -78,24 +78,32 @@ There are opportunities for developers at all levels to contribute. [Learn more
 = 4.0.0 =
 Release date: April 20th, 2016
 
-Enhancements:
+Performance Enhancements:
 
-* Contact Forms: improved performance of the routine that daily cleans the database up of form submissions flagged as spam.
-* Custom Content Types: if a testimonial entry has a featured image, the CSS class `has-testimonial-thumbnail` is now added to the div.testimonial-entry enclosing tag.
-* General: admin page before connecting Jetpack now displays information about what users receive after connecting.
-* Protect: improved performance of routine that checks whether Protect is active for a site.
-* Sharing: in WooCommerce singular product views, social icons are now placed under the add to cart button.
-* Shortcodes: the module description now includes new links to documentation pages for recently added shortcodes.
-* Theme Tools: breadcrumbs now include schema.org microdata that helps search engines like Google understand the current page’s position, where breadcrumbs are displayed, in the site hierarchy.
-* VideoPress: new view for Editor to edit the VideoPress shortcode visually like it can be done for WP galleries.
-* Widgets: they're now compatible with WP 4.5 Customizer's Selective Refresh. This allows to update a widget and, instead of reloading the entire page in preview, only its portion in preview is refreshed.
-* Widget Visibility: new options available to choose to hide or show a widget in custom post type single and archive views.
+* Protect: the routine that verifies your site is protected from brute-force attacks got some love and is more efficient.
+* Contact Forms: cleaning the database of spam form submission records is more efficient. Nothing to see here!
+
+Feature Improvements:
+
+* VideoPress: edit your VideoPress shortcode in the editor with a fancy new modal options window.
+* Custom Content Types are now classier: a new CSS class on Testimonial featured images — has-testimonial-thumbnail — allows you to customize Jetpack custom post types as you see fit.
+* Sharing: social icons are now placed under the "add to cart” singular product views in WooCommerce, making it easier for customers to share their purchases on social media.

```

Comment:
The "share their purchases" phrase might not be entirely correct since the product someone is sharing might not have been purchased.